cli/src/main.rs

```diff
@@ -160,7 +163,8 @@ fn encrypt(config: &str) -> Result<()> {
     io::stdin().read_to_end(&mut input)?;
 
     let executor = RealCommandExecutor;
-    let jwk = fetch_and_prepare_jwk(&config.servers, &config.path, initdata.clone(), &executor)?;
+    let num_retries = config.num_retries.as_ref().unwrap_or(&NumRetries::Finite(10));
```

Comment:
This 10 should be probably made into a constant; it is used again further on in `decrypt()`.